client/components/Overview/components/ImageGallery/ImageGalleryThumbnail.jsx

```diff
@@ -33,7 +33,8 @@ const ImageGalleryThumbnail = ({
                 key={idx}
                 style={(idx === currentImageIndex) ? { backgroundImage: `url(${style.thumbnail_url})`, opacity: 1 } : { backgroundImage: `url(${style.thumbnail_url})` }}
                 alt=""
-                onClick={() => {
+                onClick={(e) => {
+                  e.stopPropagation();
```

Comment:
What is this?

Reply:
If there is a parent div that has an on click (big image) and a child div with an on click (thumbnail images) click on the child component will actually trigger the parent components on click making the image enter exanded view rather than cycle to the selected thumbnail image. This stops that from happening.